samples/openapi3/client/petstore/python-pydantic-v1/tests/test_api_client.py

```diff
@@ -156,6 +155,11 @@ def test_sanitize_for_serialization(self):
         result = self.api_client.sanitize_for_serialization(data)
         self.assertEqual(result, "1997-07-16T19:20:30.450000+01:00")
 
+        # decimal
+        data = Decimal("1.0")
+        result = self.api_client.sanitize_for_serialization(data)
+        self.assertEquals(result, "1.0")
+
```

Comment:
Actually, can you move this into `samples/openapi3/client/petstore/python/tests/test_api_client.py` instead?

The "pydantic v1" backend will be removed at some point.

Reply:
oh good catch, thats on me, fixing now

Reply:
3724025e0def4b2713a85d623df7dcbb4afcfe1a

Reply:
@multani @wing328  do i need to worry about the failing steps in the build? They appear to be failures to install poetry for the v1 petstore sample. 

```
Run pipx install --python '/opt/hostedtoolcache/Python/3.7.17/x64/bin/python' poetry
creating virtual environment...
installing poetry...
Fatal error from pip prevented installation. Full pip output in file:
    /opt/pipx/logs/cmd_2024-07-22_16.55.46_pip_errors.log

Some possibly relevant errors from pip install:
    TypeError: Expected maxsize to be an integer or None
Error installing poetry.
```

Reply:
Hm, I can't retry, but I don't think it's related to your change :)